src/Features/Core/Portable/EditAndContinue/EditAndContinueWorkspaceService.cs

```diff
@@ -162,6 +157,38 @@ internal static bool SupportsEditAndContinue(Project project)
         internal static bool IsDesignTimeOnlyDocument(Document document)
             => document.Services.GetService<DocumentPropertiesService>()?.DesignTimeOnly == true;
 
+        public async Task<ImmutableArray<ImmutableArray<(LinePositionSpan, ActiveStatementFlags)>>> GetBaseActiveStatementSpansAsync(ImmutableArray<DocumentId> documentIds, CancellationToken cancellationToken)
+        {
+            var editSession = _editSession;
+            if (editSession == null)
```

Comment:
does this indicate a bug?

Reply:
No. The caller usually won't call this method unless the edit session is in progress but there can be a race (especially since this is gonna be a remote call).